priv/repo/migrations/20240605144358_contacts.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Orbit.Repo.Migrations.Contacts do
+  use Ecto.Migration
+
+  def change do
+    create table(:contacts) do
+      add(:first_name, :string)
+      add(:middle_initial, :string)
+      add(:last_name, :string)
+      add(:email_address, :string)
+      add(:badge_number, :string)
+      add(:area, :integer)
+      timestamps()
+    end
+
+    create index(:contacts, [:badge_number])
```

Comment:
should this be a unique index? In Glides I think we've been using on `on_conflict: :replace` in a couple places that would depend on it being unique.

Reply:
Oh, yes! thanks